### ISSUE TITLE
Unable to batch-set DNC on contacts

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic#8838]
## Bug Description
When selecting multiple contacts and choosing set DNC, a 500 internal server error is thrown method getLead not exists

| Q   | A
| --- | ---
| Mautic version | 3.0.0-beta-2
| PHP version | 7.2.31
| Browser | Chrome 83

### Steps to reproduce
1. Select multiple leads
2. Batch change to set DNC
 
```
 500 Internal Server Error - Method getLead not exists 

InvalidArgumentException
/app/bundles/CoreBundle/Entity/CommonEntity.php:44 at
/app/bundles/LeadBundle/Entity/CustomFieldEntityTrait.php:84 at Mautic\CoreBundle\Entity\CommonEntity -> __call ( 'getLead', array() )
/app/bundles/LeadBundle/Controller/LeadController.php:1628 at Mautic\LeadBundle\Entity\Lead -> __call ( 'getLead', array() )
/app/bundles/CoreBundle/Controller/CommonController.php:449 at Mautic\LeadBundle\Controller\LeadController -> batchDncAction ( '0', '' )
/vendor/symfony/http-kernel/HttpKernel.php:151 at Mautic\CoreBundle\Controller\CommonController -> executeAction ( 'batchDnc', '0', '0', '' )
/vendor/symfony/http-kernel/HttpKernel.php:68 at Symfony\Component\HttpKernel\HttpKernel -> handleRaw ( object(Request), '1' )
/vendor/symfony/http-kernel/Kernel.php:200 at Symfony\Component\HttpKernel\HttpKernel -> handle ( object(Request), '1', true )
/app/AppKernel.php:104 at Symfony\Component\HttpKernel\Kernel -> handle ( object(Request), '1', true )
/app/middlewares/CORSMiddleware.php:91 at AppKernel -> handle ( object(Request), '1', true )
/app/middlewares/CatchExceptionMiddleware.php:43 at Mautic\Middleware\CORSMiddleware -> handle ( object(Request), '1', true )
/app/middlewares/Dev/IpRestrictMiddleware.php:64 at Mautic\Middleware\CatchExceptionMiddleware -> handle ( object(Request), '1', true )
/app/middlewares/VersionCheckMiddleware.php:67 at Mautic\Middleware\Dev\IpRestrictMiddleware -> handle ( object(Request), '1', true )
/app/middlewares/TrustMiddleware.php:51 at Mautic\Middleware\VersionCheckMiddleware -> handle ( object(Request), '1', true )
/vendor/stack/builder/src/Stack/StackedHttpKernel.php:23 at Mautic\Middleware\TrustMiddleware -> handle ( object(Request), '1', true )
/vendor/stack/run/src/Stack/run.php:13 at Stack\StackedHttpKernel -> handle ( object(Request) )
/index_dev.php:25 at Stack\run ( object(StackedHttpKernel) )
/index.php:6 at require_once ( '/var/app/current/code/pulls/3.x/index_dev.php' )
```

**Comment by RCheesley** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Contact > Check batch update - Set Do Not Contact](https://trello.com/c/2TAF0ods/34-contact-check-batch-update-set-do-not-contact)

**Comment by kuzmany** [comment]
I'll take it.

**Comment by RCheesley** [comment]
Closing as this was fixed by #8847
